libs/extensions/angular/image-banner/src/image-banner.component.stories.ts

```diff
@@ -106,3 +106,43 @@ export const NoDismiss: Story = {
     template: `<kirby-x-image-banner ${argsToTemplate(args)}></kirby-x-image-banner>`,
   }),
 };
+
+export const containerQueries: Story = {
+  args: {
+    title: 'A title of',
+    bodyText: 'Some body text.',
+    actionButtonText: 'Read more',
+    imagePath: 'assets/images/leaves.jpg',
+    dismissClick: undefined,
+  },
+  // The render method with argsToTemplate() is needed for bannerDismiss to not be automatically inferred by storybook.
+  render: (args) => ({
+    props: args,
+    template: `
+      <div class="kirby-grid">
+        <div class="kirby-grid-item" xs="12">
+          <kirby-x-image-banner ${argsToTemplate(args)}></kirby-x-image-banner>
+        </div>
+        <div class="kirby-grid-item" xs="6">
+          <kirby-x-image-banner ${argsToTemplate(args)}></kirby-x-image-banner>
+        </div>
+        <div class="kirby-grid-item" xs="6">
+          <kirby-x-image-banner ${argsToTemplate(args)}></kirby-x-image-banner>
+        </div>
+        <div class="kirby-grid-item" xs="3">
+          <kirby-x-image-banner ${argsToTemplate(args)}></kirby-x-image-banner>
+        </div>
+        <div class="kirby-grid-item" xs="3">
+          <kirby-x-image-banner ${argsToTemplate(args)}></kirby-x-image-banner>
+        </div>
+        <div class="kirby-grid-item" xs="3">
+          <kirby-x-image-banner ${argsToTemplate(args)}></kirby-x-image-banner>
+        </div>
+        <div class="kirby-grid-item" xs="3">
+          <kirby-x-image-banner ${argsToTemplate(args)}></kirby-x-image-banner>
+        </div>
```

Comment:
Does this last row provide any value in docs or screenshot testing?
It does not showcase too well on mobile, consider just having two rows as I think it still shows your intention.

Reply:
<img width="345" alt="image" src="https://github.com/user-attachments/assets/ec7b3f5e-d688-480d-ba27-45c58db76d6f">